wildfly-elytron/pom.xml

```diff
@@ -935,6 +944,7 @@
                                     <include>org.wildfly.security.permission</include>
                                     <include>org.wildfly.security.sasl.util</include>
                                     <include>org.wildfly.security.ssl</include>
+                                    <include>org.wildfly.security.ssh.util</include>
```

Comment:
Since the existing `Pem` class was private API, I think the new utility class can also be private API so no need to add it here for now.

Reply:
fixed